composer.json

```diff
@@ -31,11 +31,11 @@
         "php": "^8.1.0",
         "ext-json": "*",
         "laravel/framework": "^10.0|^11.0",
-        "symfony/console": "^5.3|^6.0|^7.0"
+        "symfony/console": "^6.2|^7.0"
     },
     "require-dev": {
         "roave/security-advisories": "dev-master",
-        "orchestra/testbench": "^8.0|^9.0",
+        "orchestra/testbench": "^8.0|^9.2",
```

Comment:
This change is required to comply with [CVE-2024-52301 ](https://github.com/laravel/framework/security/advisories/GHSA-gv7v-rgg6-548h) enforced by `roave/security-advisories`.